File: packages/design-system/src/tokens/shape.ts
// Radius tokens mirror docs/ui/00_design-system.css custom-property values:
//   --radius-sm: 6, --radius-md: 9, --radius-lg: 13, --radius-xl: 16, --radius-full
// Names below are semantic so primitives don't have to know the numeric value.

export const radius = {
  input: 9, // var(--radius-md)
  button: 9,
  card: 13, // var(--radius-lg)
  panel: 13,
  modal: 16, // var(--radius-xl)
  pill: 9999, // var(--radius-full)
  avatar: "50%",
  checkbox: 6, // var(--radius-sm)
  codeChip: 6,
} as const;

export type RadiusTokens = typeof radius;

File: packages/design-tokens-flutter/lib/src/shape.dart
/// Radii from docs/ui/01_StudentPortal_Web/00_design-system.css:
///   --radius-sm: 6, --radius-md: 9, --radius-lg: 13, --radius-xl: 16, --radius-full
class AlpRadius {
  AlpRadius._();

  static const double input = 9; // --radius-md
  static const double button = 9;
  static const double card = 13; // --radius-lg
  static const double panel = 13;
  static const double modal = 16; // --radius-xl
  static const double pill = 9999;
  static const double checkbox = 6; // --radius-sm
  static const double codeChip = 6;

  /// Avatar uses `BoxDecoration(shape: BoxShape.circle)` in Flutter; no radius constant needed.
}
